mne/io/brainvision/brainvision.py

```diff
@@ -477,20 +477,32 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
     misc = list(misc_chs.keys()) if misc == 'auto' else misc
 
     # create montage
-    if montage is True:
-        from ...transforms import _sphere_to_cartesian
+    if cfg.has_section('Coordinates') and montage is None:
+        from ...transforms import _sph_to_cart
         from ...channels.montage import Montage
         montage_pos = list()
         montage_names = list()
+        to_misc = list()
         for ch in cfg.items('Coordinates'):
-            montage_names.append(ch_dict[ch[0]])
+            ch_name = ch_dict[ch[0]]
+            montage_names.append(ch_name)
             radius, theta, phi = map(float, ch[1].split(','))
             # 1: radius, 2: theta, 3: phi
-            pos = _sphere_to_cartesian(r=radius, theta=theta, phi=phi)
+            pol = np.deg2rad(theta)
+            az = np.deg2rad(phi)
+            pos = _sph_to_cart(np.atleast_2d([radius * 85., az,
+                                              pol])).squeeze()
+            if np.unique(pos).size == 1 and ch_name not in list(eog) + misc:
```

Comment:
`(pos == 0).all()` rather than `np.unique`?

Reply:
this is how it is checked in `mne.base._check_update_montage` which is why I choose to do so here. However, what we really want to check is all zeros so yours make more sense I guess.